elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -99,7 +72,17 @@ public EntityDictionary() {
      * @return binding class
      */
     public String getBinding(Class<?> entityClass) {
-        return bindEntityToJsonApi.get(lookupEntityClass(entityClass));
+        return entity(entityClass).jsonApi;
+    }
+
+    /**
+     * Get the Database Manager for an entity
+     *
+     * @param entityClass entity name
+     * @return List of relationship names for entity
+     */
+    public DatabaseManager getDatabaseManager(Class<?> entityClass) {
```

Comment:
I didn't think about this: good point.

I suspect we would face a similar issue within a DB manager, though. You would likely need to have a set of nested transactions and a method to rollback previously committed transactions (since someone would _have_ to commit first). For instance, some pseudo code that describes what I'm thinking:

```java
// Initial call like: 
boolean openTransactions(List<DatabaseManager> dbmanagers, List<DatabaseTransaction> txs, int level) {
    if (level == dbmanagers.size()) {
        // Do some work within the transactions
        // Also need a good way to pass the transaction types into the system.
        // Return true assuming successful action
        return true;
    }
    try (DatabaseTransaction tx = dbmanagers.get(level).getTransaction()) {
        txs.add(tx);
        if (!openTransactions(dbmanagers, txs, level + 1)) {
            tx.close(); // This is currently how this works (close before commit should rollback if necessary)
            return false; // But we can short-circuit the exception logic below by returning early
        }
        tx.commit(); // Close before commit if any other level failed
    } catch (Exception e) {
        // Here is where we would not to 
        for (int i = level + 1 ; i < dbmanagers.size() ; ++i) {
            someTxId = ...; // Just a suggestion, but we need some sort of identifier for this transaction across managers
            dbmanagers.get(i).rollbackTransaction(someTxId);
        }
        return false; // Transaction closed, error'd, etc. before commit was successful: don't let anyone else commit
    }
    return true;
}
```

This is some off-the-cuff pseudo-code, but I think we can actually solve this generally because we would still have to be managed if you encapsulated the logic in a single DB manager.